lib/Doctrine/ORM/Mapping/ClassMetadataInfo.php

```diff
@@ -19,3381 +19,10 @@
-    }
-
-    /**
-     * Checks whether the mapped class uses the JOINED inheritance mapping strategy.
-     *
-     * @return boolean TRUE if the class participates in a JOINED inheritance mapping,
-     *                 FALSE otherwise.
-     */
-    public function isInheritanceTypeJoined()
-    {
-        return $this->inheritanceType == self::INHERITANCE_TYPE_JOINED;
-    }
-
-    /**
-     * Checks whether the mapped class uses the SINGLE_TABLE inheritance mapping strategy.
-     *
-     * @return boolean TRUE if the class participates in a SINGLE_TABLE inheritance mapping,
-     *                 FALSE otherwise.
-     */
-    public function isInheritanceTypeSingleTable()
-    {
-        return $this->inheritanceType == self::INHERITANCE_TYPE_SINGLE_TABLE;
-    }
-
-    /**
-     * Checks whether the mapped class uses the TABLE_PER_CLASS inheritance mapping strategy.
-     *
-     * @return boolean TRUE if the class participates in a TABLE_PER_CLASS inheritance mapping,
-     *                 FALSE otherwise.
-     */
-    public function isInheritanceTypeTablePerClass()
-    {
-        return $this->inheritanceType == self::INHERITANCE_TYPE_TABLE_PER_CLASS;
-    }
-
-    /**
-     * Checks whether the class uses an identity column for the Id generation.
-     *
-     * @return boolean TRUE if the class uses the IDENTITY generator, FALSE otherwise.
-     */
-    public function isIdGeneratorIdentity()
-    {
-        return $this->generatorType == self::GENERATOR_TYPE_IDENTITY;
-    }
-
-    /**
-     * Checks whether the class uses a sequence for id generation.
-     *
-     * @return boolean TRUE if the class uses the SEQUENCE generator, FALSE otherwise.
-     */
-    public function isIdGeneratorSequence()
-    {
-        return $this->generatorType == self::GENERATOR_TYPE_SEQUENCE;
-    }
-
-    /**
-     * Checks whether the class uses a table for id generation.
-     *
-     * @return boolean TRUE if the class uses the TABLE generator, FALSE otherwise.
-     */
-    public function isIdGeneratorTable()
-    {
-        return $this->generatorType == self::GENERATOR_TYPE_TABLE;
-    }
-
-    /**
-     * Checks whether the class has a natural identifier/pk (which means it does
-     * not use any Id generator.
-     *
-     * @return boolean
-     */
-    public function isIdentifierNatural()
-    {
-        return $this->generatorType == self::GENERATOR_TYPE_NONE;
-    }
-
-    /**
-     * Checks whether the class use a UUID for id generation.
-     *
-     * @return boolean
-     */
-    public function isIdentifierUuid()
-    {
-        return $this->generatorType == self::GENERATOR_TYPE_UUID;
-    }
-
-    /**
-     * Gets the type of a field.
-     *
-     * @param string $fieldName
-     *
-     * @return \Doctrine\DBAL\Types\Type|string|null
-     *
-     * @todo 3.0 Remove this. PersisterHelper should fix it somehow
-     */
-    public function getTypeOfField($fieldName)
-  
```

Comment:
With `class_alias()`, `ClassMetadataInfo` is shown as unknown class in my PHPStorm IDE. I guess we can live with that since we don't code for IDEs and an error will be shown in the IDE telling us, that something important changed.

Reply:
Hm, it took the deleted side of the diff instead of the one with the class_alias. Sorry.